engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -273,13 +273,24 @@ <h4 translate>Content Host Content</h4>
         </dd>
 
         <dt translate>Content View</dt>
-        <dd bst-edit-select="host.content_facet_attributes.content_view.name"
+        <dd>
+          <div bst-edit-select="host.content_facet_attributes.content_view.name"
             readonly="denied('edit_hosts', host)"
             selector="host.content_facet_attributes.content_view.id"
             options="contentViews()"
             on-cancel="cancelContentViewUpdate()"
             on-save="saveContentView(host)"
             edit-trigger="editContentView">
+            </div>
+            <div bst-alert="info" ng-show="showCVAlert">
+              <p translate>
+                Please run the following commands on the client, so the new Content View can be picked up.
```

Comment:
"Picked up" sounds a little too folksy to me.. perhaps
```suggestion
                Please run the following commands on the client, so it is aware of its new content view.
```
```suggestion
                Please run the following commands on the client, so the new Content View can take effect.
```

Reply:
Sounds good, will add.